Libraries/Inspector/InspectorPanel.js

```diff
@@ -21,7 +21,19 @@ const Text = require('Text');
 const TouchableHighlight = require('TouchableHighlight');
 const View = require('View');
 
-class InspectorPanel extends React.Component<$FlowFixMeProps> {
+type Props = $ReadOnly<{|
+  devtoolsIsOpen?: ?boolean,
+  inspecting?: ?boolean,
```

Comment:
Circle CI errors indicate that we need to strict type it rather than `?boolean`. Do we want strict type checking or nullable type checking? or maybe both. I think this applies to all the boolean values when we are doing `if(value) then something`

Reply:
Does the following work?

```javascript
  if (value === true) {
    // stuff
  }
```

Reply:
Yes. But these props can be either boolean or undefined. I don't know how to write for both? `?boolean` should work, but I get this error:

```
Sketchy null check on boolean [1] which is potentially false. Perhaps you meant to check for null or undefined [2]?
(`sketchy-null-bool`)`
```

Reply:
Hmm... There's only one use of `<InspectorPanel>` in the codebase: 

```react
<InspectorPanel
  devtoolsIsOpen={!!this.state.devtoolsAgent}
  inspecting={this.state.inspecting}
  perfing={this.state.perfing}
  setPerfing={this.setPerfing.bind(this)}
  setInspecting={this.setInspecting.bind(this)}
  inspected={this.state.inspected}
  hierarchy={this.state.hierarchy}
  selection={this.state.selection}
  setSelection={this.setSelection.bind(this)}
  touchTargeting={Touchable.TOUCH_TARGET_DEBUG}
  setTouchTargeting={this.setTouchTargeting.bind(this)}
  networking={this.state.networking}
  setNetworking={this.setNetworking.bind(this)}
/>
```

Lets make the `inspecting`, `setInspecting`, `perfing`, `setPerfing`, `selection`, `setSelection` props non-optional. I think that's better anyways, since this way, we'll make the props more strict.